### ISSUE TITLE
accessibilityLabel missing in Touchables ?

### ISSUE THREAD

**Issue by fangmobile** [facebook/react-native#5322]
https://github.com/facebook/react-native/blob/0.14-stable/Libraries/Components/Touchable/TouchableHighlight.js#L211~L236

Not able to set accessibilityLable for touchableHilight or TouchableOpacity etc. In the code quoted above, when return the view in the render function, accessibilityLable is not there.

Even tried to put it the UIExample by adding a prop in 
https://github.com/facebook/react-native/blob/master/Examples/UIExplorer/TouchableExample.js#L132~L152
Since accessibilityLabel is not returned in the render function, it will have not effect even if I added such a prop in the example.

Is this by design?

**Comment by ide** [comment]
Good find - we should support accessibilityLabel. The accessibility attributes also should be consistent across TouchableOpacity, TouchableHighlight, TouchableNativeFeedback, TouchableBounce, and TouchableWithoutFeedback.

**Comment by fangmobile** [comment]
Is it as simple as adding that  prop in the returned view for all the touchable js mentioned? If so I can submit a  pr soon.

**Comment by ide** [comment]
Yes, I believe that's all you'd need to do. Please test it too and describe your test plan in the PR.

**Comment by fangmobile** [comment]
fixed with #5346
